examples/holistic_landmarker/android/app/src/main/java/com/google/mediapipe/examples/holisticlandmarker/OverlayView.kt

```diff
@@ -0,0 +1,212 @@
+package com.google.mediapipe.examples.holisticlandmarker
+
+/*
+ * Copyright 2024 The TensorFlow Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *             http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import android.content.Context
+import android.graphics.Bitmap
+import android.graphics.Canvas
+import android.graphics.Paint
+import android.util.AttributeSet
+import android.view.View
+import androidx.core.content.ContextCompat
+import com.google.mediapipe.framework.image.ByteBufferExtractor
+import com.google.mediapipe.tasks.vision.core.RunningMode
+import com.google.mediapipe.tasks.vision.facelandmarker.FaceLandmarker
+import com.google.mediapipe.tasks.vision.handlandmarker.HandLandmarker
+import com.google.mediapipe.tasks.vision.holisticlandmarker.HolisticLandmarkerResult
+import com.google.mediapipe.tasks.vision.poselandmarker.PoseLandmarker
+import kotlin.math.max
+import kotlin.math.min
+import kotlin.math.roundToInt
+
+class OverlayView(context: Context?, attrs: AttributeSet?) :
+    View(context, attrs) {
+    private var results: HolisticLandmarkerResult? = null
+    private var facePaint = Paint()
+    private var handPaint = Paint()
+    private var posePaint = Paint()
+
+    private var scaleFactor: Float = 1f
+    private var imageWidth: Int = 1
+    private var imageHeight: Int = 1
+
+    init {
+        initPaints()
+    }
+
+    fun clear() {
+        results = null
+        facePaint.reset()
+        invalidate()
+        initPaints()
+    }
+
+    private fun initPaints() {
+        facePaint.color =
+            ContextCompat.getColor(context!!, R.color.mp_color_primary)
+        facePaint.strokeWidth = LANDMARK_STROKE_WIDTH
+        facePaint.style = Paint.Style.STROKE
+
+        handPaint.color =
```

Comment:
If we're making both hands the same color, the R.color value doesn't need to be 'right hand'

Reply:
I think that it would be better if we separate left and right hand into two different colors, i have pushed commit to implement this.
Does this match with our expectations?

Reply:
That sounds good to me!